smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -813,7 +813,7 @@ def deviceHandler(evt) {
 		}
 	} else if (deviceInfo) {
 		if (deviceInfo.callbackUrl) {
-			sendToHarmony(evt, deviceInfo.callbackUrl)
+			sendToHarmonyCloud(evt, deviceInfo.callbackUrl)
```

Comment:
state.harmonyHubs is the new way of storing hub connection info, while state[evt.deviceId] is the old way.  Both are checked, since we need to stay backwards compatible, but state.harmonyHubs is the "new" way. We should make sure we have the ability to call either method from either piece of data.

If you use their cloud URL to make the determination instead of an IP address, you'll just need to be able to handle the case where they decide to change their cloud URL some day.